_posts/2018-06-01.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: newsletter
+title: "Newsletter #9 - Three's company
+category: newsletter
+permalink: /news/2018/06/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to Neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
```

Comment:
Add the https:// prefix, otherwise the browser thinks this is a relative link.
i.e. gitter.im/neovim -> https://gitter.im/neovim